.github/ISSUE_TEMPLATE/future_request.yaml

```diff
@@ -0,0 +1,31 @@
+name: Feature Request
+description: |
+  Report a bug encountered while using Taichi. 
+labels: 
+  - "feature request"
+body:
+  - type: textarea
+    id: problem
+    attributes:
+      label: Concisely describe the proposed feature
+      description: |
+        A clear and concise description of what you want.
+      value: |
+        > I would like to add an Apple Metal backend to the compiler so that my Macbook GPU can be utilized.
+      render: markdown
+    validations:
+      required: true
+  - type: textarea
+    id: describe
+    attributes:
+      label: Describe the solution you'd like (if any)?
+      description: |
+        A clear and concise description of what you want to achieve and implement.
+    validations:
+      required: true
```

Comment:
I'm not sure what're the differences between this one and the box above ?

Reply:
People can describe the impl detail here.!